x/txfees/types/gov.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	ProposalTypeUpdateFeeToken = "UpdateFeeToken"
+)
+
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeUpdateFeeToken)
+	govtypes.RegisterProposalTypeCodec(&UpdateFeeTokenProposal{}, "osmosis/UpdateFeeTokenProposal")
+}
+
+var _ govtypes.Content = &UpdateFeeTokenProposal{}
+
+func NewUpdateFeeTokenProposal(title, description string, feeToken FeeToken) govtypes.Content {
+	return &UpdateFeeTokenProposal{
```

Comment:
Update means add or remove?

Reply:
Probably would be good to add description that when FeeToken.PoolID = 0, FeeToken is removed.